app/views/home/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="conatiner-fluid home-page">
+    <div class="row no-gutter">
+        <div class="col-md-12">
+            <div class="d-flex align-items-center py-5">
+             <div class="container">
+              <div class="row">
+                <div class="col-12 col-lg-6 mx-auto">
+                            <h1 class="text-center header-logo">BUDGETLY</span></h1>
+                        <div class="text-center pt-5">
+                        <%= link_to "LOG IN", new_user_session_path, class: "btn btn-outline-primary py-2 mb-4 mt-2 fw-bold w-100" %>
+                        <%= link_to 'SIGN UP', new_user_registration_path, class: "btn btn-outline-primary py-2 mb-4 fw-bold w-100" %>
+                        </div>
+                        </div>
+                    </div>
+                 </div>
+            </div>
+        </div>
+    </div>
+
+</div>
```

Comment:
- [ ] Good job here 👏 In addition to what you have done, I would suggest **that you add a background color to the login button** but also **try to position both the Login and Sign up buttons at the bottom of the screen** in order to match the design given 😃  👇 

**Your design**
![image](https://user-images.githubusercontent.com/90222110/230641251-e3783e13-63b2-4a06-952b-bdf8e47350c8.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/230641308-1b314a95-46cf-469d-a549-416dc5bc7664.png)